.github/workflows/security.yml

```diff
@@ -21,7 +21,7 @@ jobs:
           if [ $(git ls-remote --heads https://github.com/az-digital/az-quickstart-scaffolding.git $AZ_TRIMMED_REF | wc -l) = 1 ]; then
             echo "SCAFFOLD_BRANCH=${AZ_TRIMMED_REF}" >> ${GITHUB_ENV}
           else
-            echo "SCAFFOLD_BRANCH=main" >> ${GITHUB_ENV}
+            echo "SCAFFOLD_BRANCH=2.6.x" >> ${GITHUB_ENV}
```

Comment:
Was thinking we could use this same pattern: https://github.com/az-digital/arizona-bootstrap/blob/76ecae23fd30fd8b1aeab34473003b71c29c60b3/.github/workflows/release.yml#L112

Reply:
But tags might put the kibosh on that.

Reply:
I think that would probably work 🤔

Reply:
Oh yeah maybe not.